### ISSUE TITLE
About the shape of data for the model.

### ISSUE THREAD

**Issue by YangYang** [JasonSWFu/Quality-Net#4]
Hi, recently I'm reading this paper. I feel this work is very outstanding, so I want to implement this, until I find this repository.
But when I want to run your code, I find I could not known about the data fed into the model, so can you share the shape of your training data and the label of the training? I will very appreciated.

**Comment by JasonSWFu** [comment]
The input shape is (None, 257) as shown here: https://github.com/JasonSWFu/Quality-Net/blob/master/Quality_net_training_Non_intrusive.py#L170   which represents (batchsize=1, number of frames, number of frequency bins=257)

**Comment by YangYang** [comment]
Ok, thank you so much!

**Comment by YangYang** [comment]
Could you share the dataset in this paper?
